packages/react-interpolate-components/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Do we need anything here aside from `lib/` ? Shouldn't these be accounted for by the `.gitignore` in the root directory? Save for `/npm-debug.log*` which should probably be `npm-debug.log*` anyways.